### DISCUSSION TITLE
Use template in new visibility feature of sections

### DISCUSSION THREAD

**Discussion by Revilo91** [home-assistant/frontend#21000] [Dashboard]
Sometimes it would be nice to be able to allow binary templates. A workaround is to create a binary_sensor and use that. But it would also be nice if you could do that right here.

![image](https://github.com/home-assistant/frontend/assets/39066472/b6d8fe86-46dd-4077-bd0c-19da71981c08)

**Comment by JeeWee65** [comment]
I agree, everywhere in setting conditions you should be able to use templates!

**Comment by luixal** [comment]
This is totally neeed!

**Comment by adec** [comment]
Couldn’t agree more. I’m assuming that template sensors are evaluated whenever values change (or on a timed basis) even if someone isn’t using the dashboard. I’m also assuming that the visibility condition is only evaluated when the dashboard is displayed so would put less load on the HA server. Although I could be completely wrong.
